.prettierignore

```diff
@@ -0,0 +1,2 @@
+**/*.ts
```

Comment:
After further consideration, I would delete this file completely, and apply formatting to all `*.ts` files in the repo.  There's only one other such file (`scripts/prettier-swagger-plugin.ts`), and I don't mind if it gets formatted.

Also, VSCode shows this UI when saving an excluded file (the highlighted `X Prettier`), which I don't think users will like:

<img width="298" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/9459391/5681e192-ace3-4e77-a180-30b65e5fa00f">